assets/svgs/currency-usd-circle.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Needed because there is no MDI icon for this.

Reply:
In another instance, I requested a new icon from the [Material Design Github site](https://github.com/Templarian/MaterialDesign/issues/new/choose), and fairly quickly someone created one!

However, it looks like modifiers on currency icons will not be supported: https://github.com/Templarian/MaterialDesign/issues/5661